docs/core/install/dependencies.md

```diff
@@ -290,6 +290,24 @@ For .NET Core apps that use the *System.Drawing.Common* assembly, you'll also ne
 > of libgdiplus by adding the Mono repository to your system. For more information,
 > see <https://www.mono-project.com/download/stable/>.
 
+### Alpine
+
+Alpine distributions require the following libraries to be installed:
+
+- icu-libs (this is not needed if globalization is disabled)
+- krb5-libs
+- libcurl
+- libintl
+- libssl1.1 (for Alpine >= 3.9) or libssl 1.0 (for older ones)
+- libstdc++
+- lttng-ust
+- numactl (optional, useful only for devices with NUMA enabled)
+- zlib
+
+For .NET Core apps that use the *System.Drawing.Common* assembly, you also need the following dependency:
+
+- libgdiplus (it is available only in the edge/testing repository)
```

Comment:
what repo is this? maybe link to it? should it state the version like the other distros?

Reply:
Alpine has repositories named by version/kind where version is 3.6, 3.7, ... 3.11 or `edge`. Edge represents the current development. Some packages stay in the edge and they are not moved into the new version once it is created. This is the case of the libgdiplus. The kind is `main`, `community` and `testing`, which are kind of subsets of the version.